dotcom-rendering/src/web/components/Card/components/ImageWrapper.tsx

```diff
@@ -85,7 +85,7 @@ export const ImageWrapper = ({
 				isHorizontalOnMobile &&
 					css`
 						${until.tablet} {
-							width: 119px;
+							width: 128px;
```

Comment:
Here we're changing the width of left/right positioned images on mobile right?

Reply:
Yes but @HarryFischer will revisit this change